tests/plugins/mentions/mentions.js

```diff
@@ -462,6 +462,30 @@
 			assert.areEqual( '<p><strong>@Anna Doe</strong></p>', editable.getData() );
 		},
 
+		// (#2008)
+		'test config options are passed to autocomplete': function() {
+			var spy = sinon.spy( CKEDITOR.plugins, 'autocomplete' ),
+				config = {
+					followingSpace: true,
+					itemsLimit: 12,
+					throttle: 120
+				};
+
+			this.createMentionsInstance( config );
+
+			spy.restore();
+
+			assert.isTrue( spy.calledWith( this.editor, sinon.match( compareConfig ) ) );
+
+			function compareConfig( val ) {
+				for ( var k in config ) {
+					assert.areEqual( config[ k ], val[ k ], 'Option: ' + k + ' should be passed to autocomplete.' );
```

Comment:
You can use `objectAssert.areEqual` or `objectAssert.areDeepEqual` methods to compare objects.

Reply:
https://github.com/ckeditor/ckeditor-dev/pull/2078#issuecomment-448494237